### ISSUE TITLE
[QA] Mayan Swift instruction error, cannot initiate transfer 

### ISSUE THREAD

**Issue by tsadovska** [wormhole-foundation/wormhole-connect#2655]
## Describe the current behavior in detail
Checked on https://preview.portalbridge.com/b1f3c5115d/. 

Steps:
1. Prepare to swap 0.3 SOL from Solana (Phantom) to tBTC on Ethereum (MetaMask)
2. Click on "Review transaction" button
3. Click on "Confirm transaction" button

Actual result: Transaction fails.
Expected result: Transaction proceeds without error. 

```
Wormhole Connect: error completing transfer Error: Simulation failed: {"InstructionError":[7,{"Custom":6024}]}
Logs:
Program ComputeBudget111111111111111111111111111111 invoke [1]
  Program ComputeBudget111111111111111111111111111111 success
  Program ComputeBudget111111111111111111111111111111 invoke [1]
  Program ComputeBudget111111111111111111111111111111 success
  Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL invoke [1]
  Program log: Create
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [2]
  Program log: Instruction: GetAccountDataSize
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 1622 of 1388256 compute units
  Program return: TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA pQAAAAAAAAA=
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program 11111111111111111111111111111111 invoke [2]
  Program 11111111111111111111111111111111 success
  Program log: Initialize the associated token account
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [2]
  Program log: Instruction: InitializeImmutableOwner
  Program log: Please upgrade to SPL Token 2022 for immutable owner support
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 1405 of 1381616 compute units
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [2]
  Program log: Instruction: InitializeAccount3
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 4241 of 1377733 compute units
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL consumed 26512 of 1399700 compute units
  Program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL success
  Program 11111111111111111111111111111111 invoke [1]
  Program 11111111111111111111111111111111 success
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [1]
  Program log: Instruction: SyncNative
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 3045 of 1373038 compute units
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program JUP6LkbZbjS1jKKwapdHNy74zcZ3tLUZoi5QNyVTaV4 invoke [1]
  Program log: Instruction: Route
  Program obriQD1zbpyLz95G5n7nJe6a4DPjpFwa5XYPoNm113y invoke [2]
  Program log: Instruction: Swap
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [3]
  Program log: Instruction: Transfer
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 4736 of 1319992 compute units
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [3]
  Program log: Instruction: Transfer
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 4645 of 1312376 compute units
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [3]
  Program log: Instruction: Transfer
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 4645 of 1304901 compute units
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program obriQD1zbpyLz95G5n7nJe6a4DPjpFwa5XYPoNm113y consumed 61753 of 1357268 compute units
  Program obriQD1zbpyLz95G5n7nJe6a4DPjpFwa5XYPoNm113y success
  Program JUP6LkbZbjS1jKKwapdHNy74zcZ3tLUZoi5QNyVTaV4 invoke [2]
  Program JUP6LkbZbjS1jKKwapdHNy74zcZ3tLUZoi5QNyVTaV4 consumed 471 of 1293173 compute units
  Program JUP6LkbZbjS1jKKwapdHNy74zcZ3tLUZoi5QNyVTaV4 success
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [2]
  Program log: Instruction: Transfer
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 4645 of 1289126 compute units
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program JUP6LkbZbjS1jKKwapdHNy74zcZ3tLUZoi5QNyVTaV4 consumed 86223 of 1369993 compute units
  Program return: JUP6LkbZbjS1jKKwapdHNy74zcZ3tLUZoi5QNyVTaV4 VkV7AgAAAAA=
  Program JUP6LkbZbjS1jKKwapdHNy74zcZ3tLUZoi5QNyVTaV4 success
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA invoke [1]
  Program log: Instruction: CloseAccount
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA consumed 2915 of 1283770 compute units
  Program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA success
  Program BLZRi6frs4X4DNLw56V4EXai1b6QVESN1BhHBTYM9VcY invoke [1]
  Program log: Instruction: InitOrder
  Program 11111111111111111111111111111111 invoke [2]
  Program 11111111111111111111111111111111 success
  Program log: AnchorError thrown in programs/swift/src/processor/init_order.rs:82. Error Code: AmountInTooSmall. Error Number: 6024. Error Message: Amount in too small.
  Program BLZRi6frs4X4DNLw56V4EXai1b6QVESN1BhHBTYM9VcY consumed 28071 of 1280855 compute units
  Program BLZRi6frs4X4DNLw56V4EXai1b6QVESN1BhHBTYM9VcY failed: custom program error: 0x1788
    at Tt (solana-CFUFmt1f-B_hZfG2S.js:7:52127)
    at async he (solana-CFUFmt1f-B_hZfG2S.js:7:50804)
    at async signAndSendTransaction (pb-index-BwMC8vQL.js:307:61387)
    at async SDKv2Signer.signAndSend (pb-index-BwMC8vQL.js:307:64375)
    at async MayanRouteSWIFT.initiate (pb-index-BwMC8vQL.js:472:469576)
    at async SDKv2Route.send (pb-index-BwMC8vQL.js:307:68515)
    at async vt (pb-index-BwMC8vQL.js:470:135480)
```

## Notes/ Attachments

![image](https://github.com/user-attachments/assets/2e7b997d-1d9b-4491-ae63-762982553273)

![image](https://github.com/user-attachments/assets/906db6b0-1f36-4042-8f6c-21360ca061e2)

https://github.com/user-attachments/assets/eefb4863-c2cf-41bc-8598-f624e80d290a

**Comment by Wesleyleung** [comment]
@tsadovska is this still reproducible or was it fixed with #2652 ?

**Comment by tsadovska** [comment]
@Wesleyleung Issue is no longer reproducible on https://portalbridge.com/. 
* Portal Bridge version: v0.2.15-8477ae26d0
* Wormhole Connect version : v^1.0.0-beta.1-development

Scenario checked: Swap 0.58 SOL from Solana (Phantom) to tBTC on Ethereum (MetaMask). Transaction completed successfully.
* Link to [Mayan Explorer](https://explorer.mayan.finance/swap/4m6Gb6JncTzrez12gKwrLQ4AQsnhkUsrAzhFHjo4gY4R2B3CNS9GQZiECKGtFcATJYVFY85CqnfykkKidXqJnxfL)
* Link to [Etherscan](https://etherscan.io/tx/0xa9745646c6d781add4cfa5cd495ac5c13a6c0d650e510f3881347b493ddb1ab4)


<img width="365" alt="image" src="https://github.com/user-attachments/assets/dfd82c64-a6ba-4cdf-ad86-bc13f95ed7bd">


Ticket is closed.
